desktop-widgets/locationinformation.cpp

```diff
@@ -156,8 +156,8 @@ void LocationInformationWidget::acceptChanges()
 	if (!ui.diveSiteCoordinates->text().isEmpty()) {
 		double lat, lon;
 		parseGpsText(ui.diveSiteCoordinates->text(), &lat, &lon);
-		currentDs->latitude.udeg = lat * 1000000.0;
-		currentDs->longitude.udeg = lon * 1000000.0;
+		currentDs->latitude.udeg = lrint(lat * 1000000.0);
+		currentDs->longitude.udeg = lrint(lon * 1000000.0);
```

Comment:
I realize that the same logic as above applies here. This could change information in our data files.